frontend_tests/node_tests/search_suggestion.js

```diff
@@ -599,6 +601,7 @@ init();
     expected = [
         "Ted",
         "pm-with:ted@zulip.com",
+        "huddle-with:ted@zulip.com",
         "sender:ted@zulip.com",
     ];
 
```

Comment:
This test is a good start, but you should also add tests to `filter.js` and `narrow.js`.